snippets/react-snippets.code-snippets

```diff
@@ -1,290 +1,224 @@
-            "aria-live='$2",
-            "aria-describedby='$3'",
-            "aria-roledescription='$4'",
-            "aria-atomic={$5}",
-        ],
-        "description":"Add `role='timer'` to indicate to assistive technologies that an element is a numerical counter listing the amount of elapsed time from a starting point or the remaining time until an end point. Add `aria-label` to provide the name of the timer. Add `aria-live` to explicitly denote a live region (Elements with the role timer have an implicit aria-live value of off). Add `aria-describedby` to indicate the idref of an element that contains additional instructions for navigating or operating this element. Add `aria-roledescription` o give the timer a more descriptive role text for screen readers to speak. Add `aria-atomic` to set whether or not the screen reader should always present the live region as a whole, even if only part of the region changes."
-    },
-    "CellAria": {
-        "prefix": [
-            "CellAria",
-        ],
-        "body": [
-            "role='cell'",
-            "aria-colspan={$1}",
-            "aria-rowspan={$2}",
-            "aria-colindex={$3}",
-            "aria-rowindex={$4}",
-            ""
-        ],
-        "description": "Adds `role='cell'` aria attributes for a cell. A cell is identified as an element in a tabular container that does not contain column or row header information. “cells” are only valid within a construct that simulates a standard data table."
-    },
-    "SwitchAria": {
-        "prefix": [
-            "SwitchAria",
-        ],
-        "body": [
-            "role='switch'",
-            "aria-checked={$1}",
-            "aria-readonly={$2}",
-            ""
-        ],
-        "description": "Use `role='switch'` on checkboxes that represent an 'on' or 'off' state. Add `aria-checked` to indicate whether component is on or off. Add `aria-required` if the field is required."
-    },
-    "SearchboxAria" : {
-        "prefix": [
-            "SearchboxAria"
-        ],
-        "body": [
-            "role='searchbox'",
-            "aria-activedescendant='$1'",
-            "aria-multiline={$2}",
-            "aria-required={$3}",
-            "aria-readonly={$4}",
-            "aria-labelledby='$5'",
-            "aria-placeholder='$6'",
-            "aria-autocomplete='$7'",
-            ""
-        ],
-        "description": "Add `role='searchbox'` to a textbox intended for specifying search criteria."
-    },
-    "SeparatorAria": {
-        "prefix": [
-            "SeparatorAria"
-        ],
-        "body": [
-            "role='separator'",
-            "aria-orientation = 'horizontal'",
-            "aria-valuemax={${1:0}}",
-            "aria-valuemin={${2:100}}",
-            "aria-valuenow={${3:50}}",
-            ""
-        ],
-        "description": "Add `role='separator'` to a HR tag. Setting role='separator' automatically sets aria-orientation = 'horizontal'."
-    },
+  "DialogAria": {
```

Comment:
Looks like this file might have gotten messed up from merging?

I think you might need to :

1. git checkout whats currently in master for snippets/react-snippets.code-snippets
2. add _only_ the list and listItem changes

Reply:
Fixed the issue. Can you review again?